stock_env.py

```diff
@@ -1,15 +1,16 @@
 import tensorflow as tf
 import numpy as np
 from preprocess import get_data
-
+from random import randint
 
 class StockEnv():
     def __init__(self,
+                 data,
```

Comment:
is this the output of preprocessing? could you update the doc string to specify?

Reply:
okay